cli/cmd/sync_v3.go

```diff
@@ -207,7 +233,29 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 	isStateBackendEnabled := sourceSpec.BackendOptions != nil && sourceSpec.BackendOptions.TableName != ""
 
 	// Read from the sync stream and write to all destinations.
-	totalResources := 0
+	totalResources := int64(0)
+
+	var remoteProgressReporter *godebouncer.Debouncer
+	if apiClient != nil {
```

Comment:
Reading this is confusing, `apiClient` might need a better name or a comment where it's created.

Reply:
Open to feedback https://github.com/cloudquery/cloudquery/pull/16229/commits/9a24f56d8638a096738406495840ffdb28cd185c